src/suparnaNJune20/DuplicateNoFromArray.java

```diff
@@ -0,0 +1,45 @@
+/*Assignment 20 : 28th Jun 2020
+ * Program 2: Find duplicate number on Integer array in Java?
+Hint: There is exactly one number is repeated in the array. You need to write a program to
+find that duplicate number. For example, if an array with length 6 contains numbers {0, 3,
+1, 2, 3}, then duplicated number is 3.*/
+package suparnaNJune20;
+
+public class DuplicateNoFromArray {
+
+	void displayArray(int[] arrayInt) {
+		System.out.println("Input:");
+		System.out.print(+arrayInt[0]);
+		for (int index = 1; index < arrayInt.length; index++) {
+			System.out.print(" , " + arrayInt[index]);
+		}
+		System.out.println("\n");
+	}
+
+	void findDuplicateNo(int[] arrayInt) {
+		int noCnt = 1;
```

Comment:
Comment is for line 20, 21, 23 and 24: 
Since the scope of both the variables are limited to out for loop,
instead of defining them twice, we can directly declare and define in the same line on line 23 and 24. as below:
boolean noFind = false;
int noCnt = 1;

Rest code looks good to me.

Reply:
changes done.